core_impl/src/test/java/io/opencensus/stats/StatsContextTest.java

```diff
@@ -132,8 +136,8 @@ public Void apply(IntervalView view) {
     StatsContext context =
         defaultStatsContext.with(
             RpcMeasurementConstants.RPC_CLIENT_METHOD, TagValue.create("myMethod"));
-    MeasurementMap measurements =
-        MeasurementMap.of(RpcMeasurementConstants.RPC_CLIENT_ROUNDTRIP_LATENCY, 5.1);
+    MeasureMap measurements =
+        MeasureMap.of((DoubleMeasure) RpcMeasurementConstants.RPC_CLIENT_ROUNDTRIP_LATENCY, 5.1);
```

Comment:
The RPC Measure constants should use the subclass types so that this cast isn't necessary, though that should be a separate change.